content/spin/designing-spin-apps.md

```diff
@@ -0,0 +1,35 @@
+title = "Designing Spin Applications"
+template = "spin_main"
+date = "2023-05-05T00:01:01Z"
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/spin/designing-spin-apps.md"
+
+---
+
+- [Built With Spin](#built-with-spin)
+- [Structuring Spin Applications](#structuring-spin-applications)
+
+## Built With Spin
+
+Spin can be used to build many different types of applications. The following, is a list of blog articles that show how different applications are built with Spin:
+- [How I Built a Like Button for My Blog with Spin](https://www.fermyon.com/blog/how-i-built-a-like-button-for-my-blog-with-spin)
+- [Building a social app with Spin (4/4): Key-Value storage and Fermyon Cloud](https://www.fermyon.com/blog/building-a-social-app-with-spin-4)
+- [Building a social app with Spin (3.5/4): Go Postgres Usage](https://www.fermyon.com/blog/building-a-social-app-with-spin-3-5)
+- [Building a social app with Spin (3/4): Post API and Spin SDK Bindings](https://www.fermyon.com/blog/building-a-social-app-with-spin-3)
+- [Building a social app with Spin (2/4): Vue.js app and Token Verification](https://www.fermyon.com/blog/building-a-social-app-with-spin-2)
+- [Building a social app with Spin (1/4): Project Setup and first API](https://www.fermyon.com/blog/building-a-social-app-with-spin-1)
+- [Serving Static Content via WebAssembly](https://www.fermyon.com/blog/serving-static-content-via-webassembly)
+- [Bots With Spin and Fermyon Cloud](https://www.fermyon.com/blog/bots-with-spin-and-fermyon-cloud)
+- [Persistent Storage in Webassembly Applications](https://www.fermyon.com/blog/persistent-storage-in-webassembly-applications)
+- [Shortlink and QR Code Generator](https://www.fermyon.com/blog/component-reuse)
+- [Build Your Own Content Management System (CMS) From a Template](https://www.fermyon.com/blog/build-you-own-cms-from-a-template)
```

Comment:
This is really more a "get started with Bartholomew" - I'd leave it off a "designing Spin applications" page.  The other Bartholomew post covers the ground here, I think, as it goes into the why and how of _building_ Bartholomew as opposed to using it.

Reply:
Sure thing.